documentation/src/components/component.ts

```diff
@@ -118,11 +175,112 @@ class ComponentElement extends LayoutElement {
                         : html``}
                     ${this.tab === TabValue.Examples
                         ? ComponentDocs.get(this.location.params.component)
-                        : ComponentApiDocs.get(this.location.params.component)}
+                        : this.renderDocs()}
```

Comment:
Since we're finding the docs in the call above, could we not pass them into the renderDocs function directly and save the additional find call below?